go.mod

```diff
@@ -1,23 +1,90 @@
 module github.com/ory/hydra-maester
 
-go 1.16
+go 1.19
```

Comment:
i see you have changed the builder images to 1.20, so why not the whole project?